src/common/QLogging.cpp

```diff
@@ -40,3 +40,5 @@ Q_LOGGING_CATEGORY(chatterinoWidget, "chatterino.widget", logThreshold);
 Q_LOGGING_CATEGORY(chatterinoWindowmanager, "chatterino.windowmanager",
                    logThreshold);
 Q_LOGGING_CATEGORY(chatterinoHighlights, "chatterino.highlights", logThreshold);
+Q_LOGGING_CATEGORY(chatterinoRecentMessages, "chatterino.recentmessages",
+                   logThreshold);
```

Comment:
Originally, logging categories were kept in alphabetic order. @pajlada (supposedly unintentionally?) broke with this tradition in 7ccf60111d5e8ba54c070fe09d926bdff5290da5 but I would suggest we keep it up for new additions.

Reply:
Realphabetized in c0a20a0